### PR TITLE
New contracts deployed to the ropsten testnet

### LINKED ISSUE #135: Work on the deployment and make sure we know what contracts we’ll point to and on what testnet

### PR DESCRIPTION
**Pull request by marcelomorgado**
### Issue link

https://github.com/tasitlabs/TasitSDK/pull/279#discussion_r266935725
https://github.com/tasitlabs/TasitSDK/issues/135

### Auto-close the issue?

Closes #135 

### Types of changes

Bug fix (non-breaking change that fixes an issue)

Breaking change (fix or feature that would cause existing functionality not to work as expected)

Technical debt (a code change that doesn't fix a bug or add a feature but makes something clearer for devs)

### Notes

I've changed the accounts (mnemonic) for fresh new ones.

### CODE REVIEW

File: packages/tasit-action/src/testHelpers/helpers.js [hunk 1]
```diff
@@ -48,38 +48,19 @@ const gasParams = {
...
+  "0x573862e2beaa8dcaf00094c7a56dcb81bcf82c83fc1bb0f9292d6cd656db45df",
+  "0xcc51b0d7bb32d222416bfd885498659a9270a1790ba0fcb1771a5fd20507ffa9",
+  "0x1fed0a74c3c287f5c93479ff5ba60e1a32974d49425bd6d596ecd0b0f77e0352",
+  "0x9f071322c7c55e5d8b7e9778788e798384371c5ec716c60614dd7db009947e18",
+  "0xa6d4b9724775bbc5541158ad69e6428af60e5f2251d0bf8fd3e8ab7efa12ee93",
+  "0xe1ba62dfb842495d59579dfca29e212ceb78818af7c5869623317fc46dd1a5bf",
+  "0xda1a8c477afeb99ae2c2300b22ad612ccf4c184564e332ae9a32f784bbca8d6b",
+  "0x633a290bcdabb9075c5a4b3885c69ce64b4b0e6079eb929abb2ac9427c49733b",
 ];
+const accounts = privateKeys.map(createFromPrivateKey);
```
Comment (pcowgill) [inline packages/tasit-action/src/testHelpers/helpers.js]: Nice change!

File: packages/tasit-contracts/truffle-config.js [hunk 2]
```diff
@@ -22,11 +22,17 @@
...
+  secretFile = fs.readFileSync(".secret.json");
+} catch (error) {
+  // Ignoring since it isn't necessary for development migrations
+}
 
 const HDWalletProvider = require("truffle-hdwallet-provider");
 
 const createInfuraProvider = (network = "mainnet") => {
-  const secretFile = fs.readFileSync(".secret.json");
+  if (!secretFile) throw Error("ERROR: The .secret.json file not found.");
```
Comment (pcowgill) [inline packages/tasit-contracts/truffle-config.js]:
```suggestion
  if (!secretFile) throw Error("ERROR: .secret.json file not found.");
```

Comment (pcowgill) [reply packages/tasit-contracts/truffle-config.js]:
```suggestion
  if (!secretFile) throw Error("ERROR: The .secret.json file was not found.");
```

Comment (pcowgill) [reply packages/tasit-contracts/truffle-config.js]: Either suggestion works

File: packages/tasit-action/src/testHelpers/helpers.js (hunk 1 above)
Comment (pcowgill) [reply packages/tasit-action/src/testHelpers/helpers.js]: Resolving this

Comment (marcelomorgado) [reply packages/tasit-action/src/testHelpers/helpers.js]: Thanks

**Pull request merged**